spec/dato/client_spec.rb

```diff
@@ -27,4 +29,35 @@
       expect(response.data.url).to be_present
     end
   end
+
+  describe "CRUD operations" do
+    let(:item_type_id) { "4404" }
+    it "can create, update and delete an item" do
+      response = client.items.create(attributes: {title: "Hello world"}, item_type_id: item_type_id)
+      item_id = response.parse["data"]["id"]
+      expect(response.code).to eq(201)
+
+      response = client.items.all(item_type_id: item_type_id)
+      expect(response.parse["data"].last["attributes"]["title"]).to eq("Hello world")
+      expect(response.code).to eq(200)
+
+      response = client.items.find(item_id: item_id)
+      expect(response.parse["data"]["id"]).to eq(item_id)
+      expect(response.parse["data"]["attributes"]["title"]).to eq("Hello world")
+      expect(response.code).to eq(200)
+
+      response = client.items.update(attributes: {title: "Hello world 2"}, item_id: item_id)
```

Comment:
some spacing is missing here

Reply:
`bundle exec standardrb --fix` is removing the spaces